types/lambda-log/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "target": "es5",
```

Comment:
What effect does this change have?

Reply:
@amcasey sorry about that, it was something leftover while tinkering with the different options